### DISCUSSION TITLE
NavLink issue

### DISCUSSION THREAD

**Discussion by aryansingh2023** [remix-run/remix#5953] [Q&A]
I am using NavLink and i'm getting an issue that when i am clicking on the one of the NavLink two link is activating at the same time.Is there any possible solution of it?
My NavLink is generating like this:
/team
/team/assets
/team/code etc.

**Comment by kiliman** [comment]
Hmm... What do you mean? Can you create a simple repro on StackBlitz or CodeSandbox?

**Comment by sergiodxa** [comment]
I imagine you're getting the NavLink with `/team` and the one with `/team/asset` or `/team/code` active at the same time (e.g. `/team` and `/team/asset` or `/team` and `/team/code`).

If that's your issue, add the `end` prop to your NavLinks, you can see the docs here https://reactrouter.com/en/main/components/nav-link#end, there's a nice table with the difference between with `end` and without `end`.
